### DISCUSSION TITLE
remove theme samples

### DISCUSSION THREAD

**Discussion by obaidalrushdi** [serenity-is/Serenity#5678] [Help]
Hi
How i can remove theme samples in core 5
i follow the instruction guide to remove but i can not find the folder in model

**Comment by edwardch** [comment]
Did you remove

```cs
//<if:ThemeSamples>
typeof(Serenity.Demo.ThemeSamples.AdminLTEController).Assembly,
//</if:ThemeSamples>
```

From `Startup.cs`?

**Reply by obaidalrushdi** [reply]
Hi,
i did not now about it before
 i removed it now and everything good 
it remove folder
Thank you very much
